src/bucket/BucketTests.cpp

```diff
@@ -81,6 +81,57 @@ countEntries(std::shared_ptr<Bucket> bucket)
     auto pair = bucket->countLiveAndDeadEntries();
     return pair.first + pair.second;
 }
+
+void
+checkBucketCount(BucketList& bl, uint32_t ledgerSeq, uint32_t level, bool isCurr)
+{
+    auto bucket = bl.getLevel(level).getCurr();
+    auto sizeOf = BucketList::sizeOfCurr;
```

Comment:
rename `sizeOf` -> `sizeOfBucket`: it's too close to `sizeof`

Reply:
Done